### PR TITLE
[T4A5][F10-B4]Chia Wei Kang

### PR DESCRIPTION
**Pull request by weikangchia**

### CODE REVIEW

**Comment by weikangchia** [comment]
Ready for review

File: src/seedu/addressbook/data/tag/Tagging.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+/**
+ * Represents a Tagging in the address book.
+ */
+public class Tagging {
+    
+    private static final String PREFIX_ADD = "+";
+    private static final String PREFIX_DELETE = "-";
+    private static final String PRINT_FORMAT = "%1$s %2$s [%3$s]";
+
+    public enum TaggingType {
```
Comment (chanjunweimy) [inline src/seedu/addressbook/data/tag/Tagging.java:14]: good job in using enum :)

File: src/seedu/addressbook/data/tag/Tagging.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import seedu.addressbook.data.person.Person;
+
+/**
+ * Represents a Tagging in the address book.
+ */
+public class Tagging {
+    
+    private static final String PREFIX_ADD = "+";
+    private static final String PREFIX_DELETE = "-";
+    private static final String PRINT_FORMAT = "%1$s %2$s [%3$s]";
```
Comment (chanjunweimy) [inline src/seedu/addressbook/data/tag/Tagging.java:12]: good job in declaring a PRINT_FORMAT template for the class to follow. :)

**Comment by chanjunweimy** [comment]
@weikangchia Some comments added. Please close the PR after reading comments. Good job! :)

**Pull request closed without merging**